com.unity.render-pipelines.universal/Runtime/Data/PostProcessData.asset.meta

```diff
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: 41439944d30ece34e96484bdb6645b55
+guid: ee973005a97ef0b4e8e4207d9609e05c
```

Comment:
This really needs to be reverted :P it'll mess up everyones projects otherwise

Reply:
Good point